src/parser/csql_lexer.l

```diff
@@ -287,6 +287,9 @@ int yybuffer_pos;
 [dD][uU][pP][lL][iI][cC][aA][tT][eE]					{ begin_token(yytext);   return DUPLICATE_; }
 [dD][aA][tT][eE]							{ begin_token(yytext);   return Date; }
 [dD][oO][uU][bB][lL][eE]						{ begin_token(yytext);   return Double; }
+[dD][oO][nN][tT]_[Rr][Ee][uU][sS][eE]_[oO][iI][dD]			{ begin_token(yytext);
```

Comment:
In the case of other items, the braces for begin_token usually start at column 73, so it would be good to match the same.